schema_based_on_diagram.sql

```diff
@@ -0,0 +1,69 @@
+CREATE TABLE patients(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(100),
+  date_of_birth DATE,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE invoices(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  total_amount DECIMAL,
+  generated_at DATE,
+  payed_at DATE,
+  medical_history_id INT,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_histories(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  admitted_at DATE,
+  patient_id INT,
+  status VARCHAR(100),
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE invoice_items(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  unit_price DECIMAL,
+  quantity INT,
+  total_price DECIMAL,
+  invoices_id INT,
+  treatment_id INT,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE treatments(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  type VARCHAR(100),
+  name VARCHAR(100),
+  PRIMARY KEY(id)
+);
+
+ALTER TABLE
+  invoices
+ADD
+  CONSTRAINT fk_medical_history    KEY (medical_history_id) REFERENCES medical_histories(id);
+
+ALTER TABLE
+  invoice_items
+ADD
+  CONSTRAINT fk_invoice_id FOREIGN KEY (invoices_id) REFERENCES invoices(id);
+
+ALTER TABLE
+  invoice_items
+ADD
+  CONSTRAINT fk_treatment_id FOREIGN KEY (treatment_id) REFERENCES treatments(id);
+
+ALTER TABLE
+  medical_histories
+ADD
+  CONSTRAINT fk_patient_id FOREIGN KEY (patient_id) REFERENCES patients(id);
+
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md) for this project, which will make your database faster, please kindly add indexes for all `FOREIGN KEYS`. For example: `CREATE INDEX ON medical_histories (patient_id);`. Please kindly fix that.